tests/cases/graphics_dynamic_buffers.amber

```diff
@@ -0,0 +1,131 @@
+#!amber
+# Copyright 2020 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHADER vertex vtex_shader_uniform GLSL
+#version 430
+
+layout(location = 0) in vec4 position;
+layout(location = 0) out vec4 frag_color;
+
+layout(set = 0, binding = 0) readonly uniform block0
+{
+    vec4 in_color;
+}; 
+
+void main()
+{
+    gl_Position = position;
+    frag_color = in_color;
+}
+END
+
+SHADER vertex vtex_shader_storage GLSL
+#version 430
+
+layout(location = 0) in vec4 position;
+layout(location = 0) out vec4 frag_color;
+
+layout(set = 0, binding = 1) readonly buffer block0
+{
+    vec4 in_color;
+}; 
+
+void main()
+{
+    gl_Position = position;
+    frag_color = in_color;
+}
+END
+
+SHADER fragment frag_shader GLSL
+#version 430
+
+layout(location = 0) in vec4 frag_color;
+layout(location = 0) out vec4 final_color;
+
+void main()
+{
+    final_color = frag_color;
+}
+END
+
+# Meaningful data is read using minimum offset alignment of 256 bytes.
```

Comment:
Similar to above comment: if this is due to the Vulkan spec, please say so. E.g. The Vulkan spec guarantees that the min offset alignment required is at most 256, and must be a power of two (probably...the spec is not 100% clear). Thus, aligning offsets to 256 should always be valid.

Reply:
done